### DISCUSSION TITLE
Add GRCm39 as Assembly track

### DISCUSSION THREAD

**Discussion by adaware-loxo** [GMOD/jbrowse-components#4298] [General] [answered]
**Is your feature request related to a problem? Please describe.**

I want to add GRCm39 as reference assembly to the existing hg19 and hg38.

**Describe the solution you'd like**

Reference from ticket #4117, could you point me to a config.json which has GRCm39 added ?

**Describe alternatives you've considered**

If config.json with GRCm39 does not exist then could you please mention what part of config json do I need to edit or add to make changes to incorporate it? Thanks

**Comment by cmdcolin** [comment]
an example of a mm39 config.json exists here (for a human vs mouse synteny demo)

https://github.com/GMOD/jbrowse-components/blob/42442fedf2c1e25f70df3cf8ec1e2d8ba5106cf2/test_data/hs1_vs_mm39/config.json#L28-L43

**Comment by cmdcolin** [comment, marked as answer]
note that this config refers directly to files on the UCSC server so it fetches data remotely.

 you could also download them to your server

other options include adding refNameAliases which would look maybe something like

```
{
      "name": "mm39",
      "sequence": {
        "type": "ReferenceSequenceTrack",
        "trackId": "mm39-ReferenceSequenceTrack",
        "adapter": {
          "type": "TwoBitAdapter",
          "twoBitLocation": {
            "uri": "https://hgdownload.soe.ucsc.edu/goldenPath/mm39/bigZips/mm39.2bit"
          },
          "chromSizesLocation": {
            "uri": "https://hgdownload.soe.ucsc.edu/goldenPath/mm39/bigZips/mm39.chrom.sizes"
          }
        }
      },
      "refNameAliases": {
        "adapter": {
          "type": "RefNameAliasAdapter",
          "location": {
            "uri": "https://hgdownload.soe.ucsc.edu/goldenPath/mm39/bigZips/mm39.chromAlias.txt"
          }
        }
      }
    }
```

**Reply by cmdcolin** [reply]
note: converted issue to discussion :) (see https://github.com/GMOD/jbrowse-components/discussions)

**Comment by adaware-loxo** [comment]
Thank you so much!
